File: Software Development Module/Week5/Service.java
/*
Service.java
Billy Cussen
10/07/2020
*/

public class Service {
    
    //Data Members
    private String branch;
    private String serviceType;
    private String day;

    //Constructor
    public Service(String branch, String serviceType){
        this.branch = branch;
        this.serviceType = serviceType;
        this.day = new String();
    }

    //Compute
    public void computeDay(){

        if(this.branch.equalsIgnoreCase("Blanchardstown")){
            if(this.serviceType.equalsIgnoreCase("Full")){
                this.day = "Monday";
            } else if(this.serviceType.equalsIgnoreCase("Interim")){
                this.day = "Tuesday";
            }else{
                this.day = "Wednesday";
            }
        } else if(this.branch.equalsIgnoreCase("Drogheda")){
            if(this.serviceType.equalsIgnoreCase("Full")){
                this.day = "Thursday";
            } else if(this.serviceType.equalsIgnoreCase("Interim")){
                this.day = "Friday";
            }else{
                this.day = "Saturday";
            }
        } 
    }

    //Getter
    public String getDay(){
        return this.day;
    }
}

File: Python Course/Software Development Module - in Python/Week5/Service.py
"""
Service.py
Billy Cussen
27/09/2020
"""
#Class Declaration
class myService:
    #Constructor
    def __init__(self, serviceType,serviceLocation):
        self.serviceType = serviceType
        self.serviceLocation = serviceLocation
    
    def computeAndGetServiceDay(self):
        if self.serviceLocation == "blanchardstown":
            if self.serviceType == "full":
                return "Your Service is on Monday"
            elif self.serviceType == "pre":
                return "Your Service is on Wednesday"
            elif self.serviceType == "interim":
                return "Your Service is on Friday"
            else:
                return "Something went Wrong here!"
        elif self.serviceLocation == "drogheda":
            if self.serviceType == "full":
                return "Your Service is on Tuesday"
            elif self.serviceType == "pre":
                return "Your Service is on Thursday"
            elif self.serviceType == "interim":
                return "Your Service is on Saturday"
            else:
                return "Something went Wrong here!"
        else:
            return "Something went Wrong here!"
        